### ISSUE TITLE
There is an error when initializing a dummy vault.

### ISSUE THREAD

**Issue by hxx-ustcer** [rchatterjee/nocrack#7]
Hi sorry to bother you,  one issue I want to report.
When initializing a dummy vault,  sometimes an error is reported.  The error is
![image](https://user-images.githubusercontent.com/49444113/78992810-c29e6180-7b6e-11ea-972f-9e8d08067a5e.png)

my code is
```
from dte.honey_vault import HoneyVault
import os
def test():
    for index in range(10000):
        vault = "./hxxtest/vault_"+str(index)+".db"
        if os.path.exists(vault):
            os.remove(vault)
        hv=HoneyVault(vault,"123456")
test()
```
I have tried for many times, the error is still reported. It is worth noting that the error is randomly reported, most of the time, the initialization is correct.

[vault_730.zip](https://github.com/rchatterjee/nocrack/files/4463908/vault_730.zip) is the dummy vault which generates the error.

Thanks.

**Comment by rchatterjee** [comment]
Yeah, this is a long-standing bug. I have seen that. I am not maintaining this code actively for a long time, so not sure how quickly I will be able to resolve this. Any help will be greatly appreciated.

**Comment by hxx-ustcer** [comment]
Thank you for your reply!
